### PR TITLE
Fix - Base category to hide

### PR DESCRIPTION
**Pull request by sinpai**
* When one category is selected, it will be disabled in another dropdown

Screens - http://take.ms/V6LnF

### CODE REVIEW

**Review by nickzhihovich — COMMENTED** [review]
Why you add this changes? If I user, I want get transactions between several categories. For exammple: 
Between `food` `laptop` and `other`. If I choose this categories in `from` you disabled in `to`?  
But it can be transactions
`food - laptop`
`food - other`
`laptop - food`
`laptop - other`
`other - food`
`other - laptop`
I think here better add multiply choose

**Comment by sinpai** [comment]
I disable only the same category, that has been selected in 'From category' dropdown. I think it doesn't block your scenarios. It's possible only to send money from one category to another. Multiply choose isn't supported.

File: app/assets/javascripts/cross_categories.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+document.addEventListener('turbolinks:load', function(){
+  $('#modal-window').on('show.bs.modal', function(e) {
+    $('#transaction_cross_categories_transactions_attributes_category_to_id').on('change', function() {
+      var self = this;
+      $('#transaction_cross_categories_transactions_attributes_category_from_id').find('option').prop('disabled', function() {
+        return this.value == self.value
+      });
```
Comment (AleksSenkou) [inline app/assets/javascripts/cross_categories.js]:
```js
var toValue = this.value
...
return this.value === toValue
```

File: app/assets/javascripts/cross_categories.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    });
+
+    $('#transaction_cross_categories_transactions_attributes_category_from_id').on('change', function() {
+      var self = this;
+      $('#transaction_cross_categories_transactions_attributes_category_to_id').find('option').prop('disabled', function() {
+        return this.value == self.value
+      });
```
Comment (AleksSenkou) [inline app/assets/javascripts/cross_categories.js]:
```js
var fromValue = this.value
...
return this.value === fromValue
```

File: app/assets/javascripts/cross_categories.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    });
+  });
+});
```
Comment (AleksSenkou) [inline app/assets/javascripts/cross_categories.js]:
```js
document.addEventListener('turbolinks:load', function(){
  $('#modal-window').on('show.bs.modal', function(e) {
    var toSelector = $('#transaction_cross_categories_transactions_attributes_category_to_id')
    var fromSelector = $('#transaction_cross_categories_transactions_attributes_category_from_id')

    toSelector.on('change', function() {
      var toValue = this.value;

      fromSelector.find('option').prop('disabled', function() {
        return this.value == toValue;
      });
    });

    fromSelector.on('change', function() {
      var fromValue = this.value;

      toSelector.find('option').prop('disabled', function() {
        return this.value == fromValue;
      });
    });
  });
});
```

**Review by AleksSenkou — APPROVED** [review]
LGTM

**Pull request merged**